aries-site/src/pages/index.js

```diff
@@ -48,21 +69,13 @@ const Index = () => {
         </HomeTiles>
         <HomeTiles>
           {topicList.map(topic => (
-            <Tile
-              pad="medium"
-              background={topic.color}
-              key={topic.color}
-              onClick={() =>
-                (window.location.href = `/${topic.name.toLowerCase()}`)
-              }
+            <Link
+              key={topic.name}
+              href={`/${topic.name.toLowerCase()}`}
+              passHref
```

Comment:
Do you want documentation here for consistency? 
`// Need to pass href because of: https://github.com/zeit/next.js/#forcing-the-link-to-expose-href-to-its-child`

Reply:
Yes, I should add it here. Good catch.